posthog/src/test/java/com/posthog/vendor/uuid/UUIDTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.posthog.vendor.uuid
+
+import java.util.UUID
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+import kotlin.test.assertNotNull
+
+internal class UUIDTest {
+    @Test
+    fun `do not generate same value`() {
+        val uuid1 = TimeBasedEpochGenerator.generate()
+        val uuid2 = TimeBasedEpochGenerator.generate()
+        assertNotNull(uuid1)
+        assertNotNull(uuid2)
+        assertNotEquals(uuid1, uuid2)
```

Comment:
really we should generate more than two 🙈 but I'm nit picking...

or at least I guess we should generate a large list and do three things

1) assert they're not too slow to generate - this is probably unnecessary 🤷 
2) assert they're unique
3) assert they sort in the order they're generated

it looks like a good source library so I'm not _Too_ bothered about us testing this here

Reply:
1. for speed there's already benchmarking https://www.baeldung.com/java-generating-time-based-uuids#3-benchmark-1 and the library has perf test as well https://github.com/cowtowncoder/java-uuid-generator/tree/master/src/test/java/perf
2 and 3 the library already does that https://github.com/cowtowncoder/java-uuid-generator/blob/4bd4b29c7ff61d7a839ee7b5f5981f8170b04621/src/test/java/com/fasterxml/uuid/UUIDComparatorTest.java#L111-L129 and there are more tests with 1m runs.
Since we are just vendoring, removing all the unnecessary bits from all the other UUID versions, and converting to Kotlin, I don't think we have to worry that much.

Reply:
Added an adapted test with at least 10k units.
It tests the sorting and look for dupes.
https://github.com/PostHog/posthog-android/pull/142/commits/95aa074399643b2c54b3e7af3183c150e78da220